### PR TITLE
[master] Fixed function so correctly returns GasLimit

### PR DESCRIPTION
**Pull request by lucac-zilliqa**
I removed the line that caused this API call to constantly return "0x1" as the GasLimit of a block.

### CODE REVIEW

File: src/libServer/EthRpcMethods.cpp [hunk 1]
```diff
@@ -2240,8 +2240,6 @@ Json::Value EthRpcMethods::GetBlockDetails(const uint64_t blockNumber) {
   uint128_t fees = (isVacuous ? 0 : txBlock.GetHeader().GetRewards() * EVM_ZIL_SCALING_FACTOR);
   auto jsonBlock = GetEthBlockCommon(txBlock, false);
 
-  jsonBlock["gasLimit"] = "0x1";
```
Comment (JamesHinshelwood) [inline src/libServer/EthRpcMethods.cpp:2243]: @n-hutton Why was this here? It looks suspiciously intentional.

Comment (lucac-zilliqa) [reply src/libServer/EthRpcMethods.cpp:2243]: I was wondering that as well. I sent Nathan a message on Slack asking about this. `GetEthBlockCommon` is called in this top level function in which it then calls `JSONConversion::convertTxBlocktoEthJson` which contains this line `retJson["gasLimit"] = (boost::format("0x%x") % txheader.GetGasLimit()).str();`. It seems then that it should normally return a JSON object containing a "gasLimit" field without the line I removed.

Comment (lucac-zilliqa) [reply src/libServer/EthRpcMethods.cpp:2243]: @n-hutton responded and said that it was to prevent a division by zero error as running it in isolated server mode it would default to 0 and UI would break due to a division by 0 error

Comment (n-hutton) [reply src/libServer/EthRpcMethods.cpp:2243]: Yup, the otterscan UI wouldn't even render which took some digging to find out why. The fix looks good though!

**Review by n-hutton — APPROVED** [review]
